leakcanary-android-core/src/main/java/leakcanary/DefaultOnHeapAnalyzedListener.kt

```diff
@@ -71,6 +89,44 @@ class DefaultOnHeapAnalyzedListener(private val application: Application) : OnHe
     )
   }
 
+  /**
+   * Android TV devices do not have notifications, therefore the only easy and non-invasive way
+   * to communicate with user is via Toast messages. These are used just to grab user attention and
+   * to direct them to Logcat where a much more detailed report will be printed.
+   */
+  private fun showToast() {
+    // Post the Toast into main thread and wrap it with try-catch in case Toast crashes (it happens)
+    handler.post {
+      try {
+        Toast.makeText(
+            application,
+            "Analysis complete, please check Logcat",
```

Comment:
- R.string?
- People using a dev app might not be developers, e.g. sales or QA. So generally it's worth providing a bit more context, at least I've tried going in that direction. You probably want something that says "Heap Analysis performed, 3 new leaks, 0 library leaks (ie copy paste the language used in the notification), see details in Logcat"

In other words tell them it's useful to see logcat.